packages/core/react/__mocks__/@solana-mobile/wallet-adapter-mobile.ts

```diff
@@ -1,4 +1,4 @@
-import type { WalletName } from '@solana/wallet-adapter-base';
+import { type WalletName } from '@solana/wallet-adapter-base';
```

Comment:
🧐

Reply:
It was getting annoying to have two imports for the same dependencies everywhere. 😅

Reply:
It was getting annoying to have two imports for the same dependencies everywhere. 😅